backends/rapidpro/msg.go

```diff
@@ -478,6 +491,12 @@ func writeExternalIDSeen(b *backend, msg courier.Msg) {
 // Our implementation of Msg interface
 //-----------------------------------------------------------------------------
 
+type embeddedAttachment struct {
```

Comment:
these won't be supported in case we write messages from the spool?
Can we make these serializable to JSON and support that?

Reply:
hmm good point...

Reply:
https://en.wikipedia.org/wiki/Data_URI_scheme